src/mdm-api-properties.model.ts

```diff
@@ -1,4 +1,22 @@
 /*
+Copyright 2020-2021 University of Oxford
```

Comment:
A lot (if not all) of the source files have these duplicated licence headers, these need to be cleaned up.